tests/datadriven/test_variants.py

```diff
@@ -84,19 +152,52 @@ def verifyVariantsEqual(self, gaVariants, pyvcfVariants):
                     self.assertEqual(str(alt1), str(alt2))
             else:
                 self.assertEqual(gaVariant.alternateBases, alt)
-            # TODO check INFO fields
 
-    def testSearchAllVariants(self):
-        allVariants = []
+            pyvcfCallMap = {}
+            for call in pyvcfVariant:
+                pyvcfCallMap[call.sample] = call
+            _verifyVariantCalls()
+
+    def _verifyGaVariantsSample(self, gaVariants, sampleIds):
+        for variant in gaVariants:
+            self.assertEqual(len(variant.calls), len(sampleIds))
+            for call in variant.calls:
+                self.assertIn(call.callSetName, sampleIds)
+
+    def _verifyVariantsCallSetIds(self, searchVariants, searchsampleIds):
+        """
+        SearchVariants = None, this will get all variants
+        If searchsampleIds = None, this will get all samples
+        else if = [of something], this will iterate through the list
```

Comment:
oh, I see. `searchVariants` and `searchsampleIds` *(fix the camelCasing on that last one)* expect arrays or `None`.
Would it be simpler just to make the signature:
```
def _verifyVariantCallSetIds(self, searchVariants = [], searchSampleIds = [])
    """
    Leaving searchVariants empty will get all variants,
    leaving searchSampleIds will get all samples.
    """
```